tests/dummy/config/environment.js

```diff
@@ -18,6 +18,10 @@ module.exports = function(environment) {
       // when it is created
     },
 
+    'ember-metrics': {
```

Comment:
:+1: Yup, thanks for that. This addon was originally extracted from a client project where they required lazy initialization of analytics with dynamic keys, so it's still relevant for us to support that use case